util.go

```diff
@@ -341,12 +341,21 @@ type zeroable interface {
 // IsZero returns true when the value passed into the function is a zero value.
 // This allows for safer checking of interface values.
 func IsZero(data interface{}) bool {
+	v := reflect.ValueOf(data)
+	// check for nil data
+	switch v.Kind() {
+	case reflect.Interface, reflect.Ptr:
```

Comment:
How about also checking for `reflect.Slice` and removing the (now duplicated) check below then?

Reply:
changed, thanks!